src/options/options.html

```diff
@@ -148,6 +141,16 @@ <h2 data-i18n="__MSG_subtitleMisc__">Miscellaneous settings</h2>
 						</div>
 					</li>
 
+					<li>
+						<div id="messageContrast" aria-label="info message" data-i18n data-i18n-aria-label="__MSG_ariaMessageInfo__" class="message-box info invisible fade-hide">
```

Comment:
also better surround this thing with `.message-container` CSS class in case this may later be needed. (currently does not matter AFAIK, but well… :woman_shrugging: :wink: )

Let me know, if this does not work. :smile: